javatests/arcs/android/storage/database/DatabaseImplTest.kt

```diff
@@ -1196,6 +1205,20 @@ class DatabaseImplTest {
         // Check unused primitive values have been removed, only those used in entity are present.
         assertThat(readTextPrimitiveValues()).containsExactly("def")
         assertThat(readNumberPrimitiveValues()).containsExactly(123L, 789L)
+
+        // Check the corrent clients were notified.
+        collectionClient.eventMutex.withLock {
+            assertThat(collectionClient.deletes)
+                .containsExactly(null)
```

Comment:
Nit: this can go on the one line.

Same for the ones below.

Reply:
done